### ISSUE TITLE
Neutral Suggestion: Make this more unopinionated - (Personally, I have no strong feelings either way)

### ISSUE THREAD

**Issue by j-mcavoy** [EnterpriseQualityCoding/FizzBuzzEnterpriseEdition#475]
Fizz and Buzz are hard-coded to occur at multiples of 3 and 5, respectively. Perhaps it will be better to make these values unopinionated to keep options open in the future for other applications.

Perhaps we are making Fizz and Buzz too constrained here too, why should we decide for other developers that Fizz and Buzz _must_ be the default strings that are printed just because it happens to work with _our_ particular workflow?

Perhaps we should consider removing any deliberate choices made in this project and leave it to other developers to implement how _they_ want this program to work for _their_ application.

Of course these are hardly suggestions, as I have no strong feelings on the matter either way. I, for one, avoid forming any opinions because I wouldn't dare hinder my future self by doing something as reckless as making a decision.

**Comment by Tylersuard** [comment]
I'm chiming in here too, to say I also have no opinion in this matter.

**Comment by bmwiedemann** [comment]
This is a great idea. It would transform FBEE into a generic framework that could be used to do FizzBuzz, but could be used to solve so many other important problems, such as '99 bottles of beer'.

**Comment by jonaskoelker** [comment]
This appears to overlap in functionality with #522 

Let's hold a meeting to decide whether to mark #522 as "duplicate of #475" or "depends on #475".

**Comment by jonaskoelker** [comment]
I of course have no opinion on the matter.
